frontend/src/App.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from "react";
+
+import Header from "./Components/Headers/Header";
+import Products from "./Components/Products/Products";
+import Items from "./Components/Items/Items";
```

Comment:
If your components are all called `Components/Foo/index.tsx` (which we generally do in our codebases) then importing them becomes much easier. 

```suggestion
import Header from "./Components/Headers";
import Products from "./Components/Products";
import Items from "./Components/Items";
```

Reply:
done